### DISCUSSION TITLE
How did you come up with the name

### DISCUSSION THREAD

**Discussion by Rayfnm** [1RB/Authworria#26] [Q&A] [answered]
I think the name is pretty unique and catchy, so how did you come up with it? Is there any meaning behind it or is it just a word you liked the sound of?

**Comment by 1RB** [comment, marked as answer]
So I was scrolling through pinterest, looking at words that describe very specific feelings and i came across 4 words that i really liked, sorry i cant rememeber the other words, I then built separate demo websites and logos to see which one worked better and ended up sticking with authworria

***Authworria /Awth-worr-ea/ - The growing anxiety while completing a story before its fast-approaching deadline.***
>> **Auth**worria is derived from author and worry.  The suffix -ia represents “relating to.”
